src/app/components/Skills/SkillsWrapper.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+interface SkillsWrapperProps{
+    children: React.ReactNode;
+    as?: React.ElementType;
+}
+
+const SkillsWrapper: React.FC<SkillsWrapperProps> = ({children, as:AsComponent="div"}) => {
+    return(
+        <AsComponent className="p-16 flex items-center justify-between max-[960px]:flex-col max-w-7xl my-0 mx-auto w-full sm:w-[9/10] lg:w-[4/5]" >{children}</AsComponent>
```

Comment:
```suggestion
        <AsComponent className="p-16 flex items-center justify-between max-lg:flex-col max-w-7xl my-0 mx-auto w-full sm:w-[9/10] lg:w-[4/5]" >{children}</AsComponent>
```

Also, where is `max-w-7xl my-0 mx-auto w-full sm:w-[9/10] lg:w-[4/5]` coming from? Did something occur that caused you to add this?

Reply:
That's just the stying that I copied over from the Container component to get the correct formatting because it kept overriding the other styles when I tried importing and using the Container component.